src/status_im2/contexts/wallet/common/utils.cljs

```diff
@@ -33,3 +32,9 @@
   [number-of-accounts]
   (let [path (get-derivation-path number-of-accounts)]
     (format-derivation-path path)))
+
+(defn get-initials
```

Comment:
we have the function 
<img width="321" alt="Screenshot 2023-11-08 at 19 06 51" src="https://github.com/status-im/status-mobile/assets/22799766/fa5d6b3b-fab0-46a5-bf12-ffc63513b77c">
in src/utils/string.cljs
would that work instead?